tests/kano_apps/test_AppData.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+import json
+import os
+from mock import patch
+
+from kano_apps.AppData import get_applications
+
+TEST_APP = {
+    "title": "Test App",
+    "tagline": "Test application",
+    "description": "This is a test application to, well, test things.",
+    "slug": "test-app",
+
+    "icon": "test",
+    "colour": "#F4A15D",
+
+    "categories": ["code"],
+
+    "overrides": ["blacklisted.app"],
+
+    "packages": [],
+    "dependencies": [],
+    "launch_command": "echo 'Hi''"
+}
+
+# Blame Google translate...
+TEST_APP_ES = {
+    "title": "Pruebas",
+    "tagline": "Aplicación de prueba",
+    "description": "Esta es una aplicación de prueba para, así, las cosas de la prueba.",
```

Comment:
It might be wise to have a test case (either modify this one or create another) which explicitly uses unicode characters.